handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java

```diff
@@ -1356,19 +1358,19 @@ private void setClientAuth(ClientAuth mode) {
     }
 
     @Override
-    public void setEnableSessionCreation(boolean b) {
+    public final void setEnableSessionCreation(boolean b) {
         if (b) {
             throw new UnsupportedOperationException();
         }
     }
 
     @Override
-    public boolean getEnableSessionCreation() {
+    public final boolean getEnableSessionCreation() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) A getX() method which returns a boolean should be named isX() [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3ABooleanGetMethodName)